sihl/src/contract_email.ml

```diff
@@ -2,11 +2,12 @@ type t =
   { sender : string
   ; recipient : string
   ; subject : string
-  ; text : string
-  ; html : string option
+  ; text : string [@opaque]
```

Comment:
I think this is too opinionated, some sihl users might want to print these

Reply:
My thinking was to be secure by default. The type is public so the content can be printed separately if that is clearly intended.

Reply:
Good idea, good concept